crates/engine/primitives/src/lib.rs

```diff
@@ -114,6 +117,19 @@ pub trait PayloadValidator: fmt::Debug + Send + Sync + Unpin + 'static {
 
 /// Type that validates the payloads processed by the engine.
 pub trait EngineValidator<Types: EngineTypes>: PayloadValidator {
+    /// Validates the execution requests according to EIP-7685.
+    ///
+    /// This ensures that:
+    /// 1. The requests array is not empty
+    /// 2. Each request's ID is unique
+    /// 3. Each request's ID is non-zero
```

Comment:
these docs need to be moved to the impl for `EthereumEngineValidator`, and removed here as this is not the case for op